crates/league-toolkit/src/core/wad/chunk.rs

```diff
@@ -16,6 +17,12 @@ pub enum WadChunkCompression {
     ZstdMulti = 4,
 }
 
+impl fmt::Display for WadChunkCompression {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{:?}", self)
```

Comment:
can we display the actual enum names instead of the u8?

Reply:
For sure, didn't realize